src/status_im/ens/core.cljs

```diff
@@ -29,6 +31,26 @@
     username
     (stateofus/subdomain username)))
 
+(handlers/register-handler-fx
+ :update-ens-tx-state
+ (fn [{:keys [db]} [_ new-state username custom-domain? tx-hash]]
+   {:db (assoc-in db [:ens/registrations tx-hash] {:state new-state
+                                                   :username username
+                                                   :custom-domain? custom-domain?})}))
+
+(handlers/register-handler-fx
+ :update-ens-tx-state-and-redirect
+ (fn [cofx [_ new-state username custom-domain? tx-hash]]
+   (fx/merge cofx
+             (re-frame/dispatch [:update-ens-tx-state new-state username custom-domain? tx-hash])
+             (re-frame/dispatch [::redirect-to-ens-summary]))))
+
+(handlers/register-handler-fx
+ :clear-ens-registration
+ (fn [{:keys [db]} [_ tx-hash]]
+   (let [registrations (get db :ens/registrations)]
```

Comment:
you can also use
`(:ens/registrations db)`
it's a bit more idiomatic, but `(get db ...)` is also fine

There's also actually `(db :ens/registrations)` :) which is the fastest but we don't use it so often, both `keyword` and `map` implement `IFn` https://clojure.github.io/clojure/javadoc/clojure/lang/IFn.html

Reply:
you can also do the whole thing in one go
`{:db (update db :ens/registrations dissoc tx-hash)}`